gateway/src/main/kotlin/Event.kt

```diff
@@ -224,11 +231,25 @@ public sealed class Event {
                         DiscordUpdatedEmojis.serializer()
                     ), sequence
                 )
-                "GUILD_INTEGRATIONS_UPDATE" -> GuildIntegrationsUpdate(
+                "INTEGRATION_CREATE" -> GuildIntegrationCreate(
```

Comment:
oh wait, since this is here i wonder if the event was even received before

if not then the breaking change is fine i think, since it didn't work before anyway

Reply:
Ah yes, this confused me for a bit until i found this. The event certainly wasn't being received as my bots logs are filled with "unknown event INTEGRATION_..."